### ISSUE TITLE
Integrate VFPU math functions away from libpspmath

### ISSUE THREAD

**Issue by MotoLegacy** [nzp-team/vril-engine#4]
This way we can `inline` them all and avoid useless `jmp`s in the binary

**Comment by Peter0x44** [comment]
Another approach could be building with LTO, which would let GCC inline them.

**Comment by MotoLegacy** [comment]
Yeah this issue is old and my stance has changed a bit. That being said as of last night I helped integrate the library into the `psp-pacman` repo, so we can stop redistributing the source in this repository.
